lib/seahorn/BvOpSem2FatMemMgr.cc

```diff
@@ -224,15 +278,27 @@ class FatMemManager : public OpSemMemManager {
   FatMemValTy storeIntToMem(Expr _val, FatPtrTy ptr, FatMemValTy mem,
                             unsigned byteSz, uint64_t align) override {
     return mkFatMem(
-        m_mem.storeIntToMem(_val, mkRawPtr(ptr), mkRawMem(mem), byteSz, align));
+        m_main.storeIntToMem(_val, mkRawPtr(ptr), mkRawMem(mem), byteSz, align),
+        m_main.storeIntToMem(0, mkRawPtr(ptr), mkSlot0Mem(mem), byteSz, align),
```

Comment:
this is not needed, or is optional, I don't know whether it is cheaper to clear unused slots or let them keep their old value.  At any rate, the number of bytes being written in the slots in definitely not `byteSz` and `align` should not matter (i.e., the store is always aligned and always one word).

Reply:
ack. let the old value of slot0 and slot1 persist.